_generated/MavenAuthenticateV0/taskJsonOverride.json

```diff
@@ -0,0 +1,45 @@
+{
+    "inputs": [
+    {
+      "name": "workloadIdentityServiceConnection",
+      "label": "'Entra Workload ID-backed Azure DevOps user' Service Connection",
+      "helpMarkDown": "If this is set, feedUrl is required.",
+      "type": "connectedService:workloadidentityuser",
+      "required": false,
+      "properties": {
+        "EditableOptions": "False",
+        "MultiSelectFlatList": "False"
+      },
+      "groupName": "feedAuthentication"
+    },
+    {
+      "name": "feedUrl",
+      "label": "Azure Artifacts Feeds url.",
+      "helpMarkDown": "If this is set, workloadIdentityServiceConnection is required.",
```

Comment:
Consider improving the markdown to inform users if any other input will be ignored, same for workloadIdentityServiceConnection above.

Reply:
I added more to the message. I think it's clearer now